src/Tizen.Content.Download/Tizen.Content.Download/CacheManager.cs

```diff
@@ -0,0 +1,162 @@
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public void ResetAllCache()
+        {
+            int ret = Interop.Download.ResetAllDownloadCache();
+            if (ret != (int)DownloadError.None)
+            {
+                DownloadErrorFactory.ThrowException(ret, "Failed to reset all cache");
+            }
+        }
+
+        /// <summary>
+        /// Path of the cache stored.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public string CachePath
+        {
+            get
+            {
+                string cachePath;
+                int ret = Interop.Download.GetDownloadCachePath(out cachePath);
+                if (ret != (int)DownloadError.None)
+                {
+                    DownloadErrorFactory.ThrowException(ret, "Failed to get cache path");
+                }
+                return cachePath;
+            }
+            set
+            {
+                int ret = Interop.Download.SetDownloadCachePath(value);
+                if (ret != (int)DownloadError.None)
+                {
+                    DownloadErrorFactory.ThrowException(ret, "Failed to set cache path");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Life cycle of the cache.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when it is failed due to an invalid operation.</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when a permission is denied.</exception>
+        /// <exception cref="NotSupportedException">Thrown when not supported.</exception>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public uint CacheLifeCycle
```

Comment:
please provide the meaning of those values in documentation, or even better introduce an enum instead of using int/uint

Reply:
Lifecycle value is in 'seconds' so it can have a range of values possible. So, declaring enum doesn't seems feasible to me.

Will remarks
 <summary>
'Lifecycle of the cache in seconds.
 </summary>

<remarks>
The time after which the cache will be invalid. New requests corresponding to the cache entry need to fetch the data from server and update the cache for future requests to be served from cache.
</remarks>

Reply:
Updated to " Life cycle of the cache in seconds (Unix time). The time after which the stored cache will be invalid/stale"